### ISSUE TITLE
Build functional Slideover Side Navigation with Materialize

### ISSUE THREAD

**Issue by ka05** [ka05/tiger-books#7]
Mimick the google play / gmail side navigation drawers

**Comment by coffeeScripter** [comment]
could you draw a picture? also do you mean the mobile version or what? I are confuzzled. If you mean what I think you mean then definitely works. My bank does a similar thing too that I think you will like the look of. I can show you it later when we meet on thursday

**Comment by ka05** [comment]
Yes, the mobile version ( the user icon, background wallpaper and stuff )

**Comment by ka05** [comment]
![unnamed](https://cloud.githubusercontent.com/assets/8964925/9424439/bfa4f5ca-48ba-11e5-83a9-2fb19a900e8f.png)

Heres a screenshot of what im referring to
